portal/src/main/webapp/js/src/cbio-util.js

```diff
@@ -478,6 +478,16 @@ cbio.util = (function() {
         return target;
     }
     
+    //Get hotspot description. TODO: add type as parameter for different source of hotspot sources.
```

Comment:
instead of defining a utility function with a parameter, we can also define a template